### ISSUE TITLE
The standalone Dockerfile does not work.

### ISSUE THREAD

**Issue by AlemSnyder** [duckietown/gym-duckietown#257]
https://github.com/duckietown/gym-duckietown/blob/a6df727604bbe69dff0a50ba2a375f66e1c730c5/docker/standalone/Dockerfile#L6

I think there is supposed to be a backslash, or maybe this should be a requirements file.

**Comment by adleris** [comment]
Commit #273 fixes that layer in the compilation by adding in the '/'.

After fixing that, I had some other issues getting compilation working. I then get 

```
ERROR [6/6] RUN python3 -c "from gym_duckietown import *"
...
ModuleNotFoundError: No module named 'gym_duckietown'
```

Skipping that layer, I tried running `python3 manual_control.py`, and I had some other issues I couldn't resolve.

1. I had `ModuleNotFoundError: No module named 'gym'`, which then worked when I added a `pip3 install gym`.
2. After installing that, I had `pyglet.canvas.xlib.NoSuchDisplayException: Cannot connect to "None"` 

I've tried looking through some previous versions of the `docker/standalone/Dockerfile` and had other issues getting them to build, so I stopped my investigation.
